pwndbg/gdblib/arch.py

```diff
@@ -2,6 +2,7 @@
 
 import gdb
 import pwnlib
+import pwnlib.context
```

Comment:
?

Reply:
Because in `update` function we are using `pwnlib.context.context.arch` technically we should import `pwnlib.context` but if you want I can remove it

Reply:
We don't need that import since `pwnlib` import provides access to `pwnlib.context`

```
In [1]: import pwnlib

In [2]: pwnlib.context
Out[2]: <module 'pwnlib.context' from '/usr/local/lib/python3.10/dist-packages/pwnlib/context/__init__.py'>
```

Reply:
fixed!